kythe/cxx/indexer/cxx/KytheGraphObserver.h

```diff
@@ -285,7 +285,7 @@ class KytheGraphObserver : public GraphObserver {
 
   void recordUserDefinedNode(const NodeId& node,
                              const llvm::StringRef& node_kind,
-                             Completeness completeness) override;
+                             const absl::optional<Completeness> completeness) override;
```

Comment:
top-level "const" is meaningless on function declarations and should be omitted:

https://abseil.io/tips/109